test/sdf/world_joint_child_world.sdf

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" ?>
+<sdf version="1.10">
+  <world name="world_joint_child_world">
+    <!--
+      For ease of unittesting unique values,
+      each model's pose is displaced along the z-axis,
+      frames are displaced along the y-axis,
+      and joints are displaced along the x-axis.
+    -->
+    <model name="parent_model">
+      <pose>0 0 1 0 0 0</pose>
+      <link name="L"/>
+    </model>
+    <frame name="child_frame" attached_to="world">
+      <pose>0 1 0 0 0 0</pose>
+    </frame>
+    <joint name="J1" type="ball">
+      <pose>1 0 0 0 0 0</pose>
+      <parent>parent_model::L</parent>
+      <child>child_frame</child>
+    </joint>
+    <joint name="J2" type="ball">
+      <pose>10 0 0 0 0 0</pose>
+      <parent>parent_model::L</parent>
+      <child>world</child>
```

Comment:
Can you add a comment that this is an invalid file?

Reply:
I renamed the file to include `invalid` in https://github.com/gazebosim/sdformat/pull/1117/commits/d8b0e2dc5748e0672fa7d90021f846eceb6f705e